portal-web/test/functional/com/liferay/portalweb/paths/portlet/frontenddataset/FrontendDataSetAdmin.path

```diff
@@ -228,6 +228,11 @@
 	<td>//table[contains(@class,'table-list')]//tr[@class='orderable-table-row'][${key_tableRow}]//td[contains(.,'${chosenFilter}')][${key_tableColumn}]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>SELECT_ANY</td>
```

Comment:
This name is too general. What are you selecting here? Datasets? Then call it SELECT_DATASET :)

Reply:
I'll made